packages/react/src/components/Slider/Slider.tsx

```diff
@@ -842,6 +842,7 @@ export default class Slider extends PureComponent<SliderProps> {
                 <input
                   type={hideTextInput ? 'hidden' : inputType}
                   id={`${id}-input-for-slider`}
+                  data-testid="slider-input-id"
```

Comment:
Could we get this via `getByRole('textbox', name: 'whatever the name is')` instead? I'd prefer not to add a `data-testid` to the component source